controllers/substitution.go

```diff
@@ -0,0 +1,43 @@
+package controllers
+
+import (
+	v1 "k8s.io/api/apps/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// Names in that list are unique and will be substituted with an image from config
```

Comment:
is this an orphan comment?

Reply:
No, I think this comment is in place. This is planned to be a list later. Things like `kube-rbac-proxy` and `node-cloud-controller-manager` should belong here as well.

Reply:
ok, cool. i just wanted to make sure. i have a small suggestion here
```suggestion
	// Names in this list are unique and will be substituted with an image from config
```